include/dt-bindings/pinctrl/mchp-xec-pinctrl.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2021 Microchip Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_MCHP_XEC_PINCTRL_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_MCHP_XEC_PINCTRL_H_
+
+#include <dt-bindings/pinctrl/mchp-xec-pinctrl-common.h>
+
+/* Adapted from Linux: include/dt-bindings/pinctrl/stm32-pinfunc.h */
+
+/**
+ * @brief Pin modes
+ */
+
+/*
+ * Microchip XEC GPIO pins implement up to 8 functions.
+ * Function 0 is GPIO mode.
+ */
+#define MCHP_XEC_AF0		0x0
```

Comment:
suggest an enum

Reply:
DTC error if these are converted to a C enum.  I found issue 21273 where someone wanted to use enum's for USB settings in device tree. Do not know if its resolved.